libs/core/type_support/include/hpx/type_support/uninitialized_relocate_n_primitive.hpp

```diff
@@ -14,13 +14,13 @@
 #include <cstring>
 #include <type_traits>
 
-#if defined(HPX_HAVE_P1144_STD_RELOCATE_AT)
+#if defined(__cpp_lib_trivially_relocatable)
```

Comment:
I'd suggest leaving this as a feature test instead of relying on a library supplied macro. Please feel free however to use this macro as part of the feature test...

Reply:
Okay, on that note, in case P1144 is present, would it make sense to remap our type_support primitives to the sequential relocation functions in the std? (`std::uninitialized_relocate`, `std::uninitialized_relocate_n`, `std::uninitialized_relocate_backward`)?

Reply:
Yes, we usually simply import the standard versions into namespace hpx, thus avoiding changes to other code.

Reply:
Reverted this and mapped `uninitialized_relocate(_n, _backward)_primitive` --> `uninitialized_relocate(_n, _backward)` in https://github.com/STEllAR-GROUP/hpx/pull/6364/commits/2474a61c5f6f32063de5b052df3e4e7c7e2b285f